test/unit-test/signaling_api/signaling_api_utest.c

```diff
@@ -75,3 +75,1168 @@ void test_signaling_ConstructDescribeSignalingChannelRequest_BadParams( void )
+ */
+void test_signaling_ParseDescribeMediaStorageConfigResponse_InvalidJson( void )
+{
+    const char * message = "{\"MediaStorageConfiguration\": {";  /* Missing closing brace */
+    size_t messageLength = sizeof( message );
+    SignalingMediaStorageConfig_t mediaStorageConfig;
+    SignalingResult_t result;
+
+    result = Signaling_ParseDescribeMediaStorageConfigResponse( message,
+                                                                messageLength, &( mediaStorageConfig ) );
+
+    TEST_ASSERT_EQUAL( SIGNALING_RESULT_INVALID_JSON,
+                       result );
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Validate Signaling Parse Describe Media Storage Config functionality.
+ */
+void test_signaling_ParseDescribeMediaStorageConfigResponse_UnexpectedResponse( void )
+{
+    const char * message = "{\"UnexpectedKey\": {}}";        /* Unkown Key */
+    size_t messageLength = strlen( message );
+    SignalingMediaStorageConfig_t mediaStorageConfig;
+    SignalingResult_t result;
+
+    result = Signaling_ParseDescribeMediaStorageConfigResponse( message,
+                                                                messageLength, &( mediaStorageConfig ) );
+
+    TEST_ASSERT_EQUAL( SIGNALING_RESULT_UNEXPECTED_RESPONSE,
+                       result );
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Validate Signaling Parse Describe Media Storage Config functionality.
+ */
+void test_signaling_ParseDescribeMediaStorageConfigResponse_MissingFields( void )
+{
+    SignalingMediaStorageConfig_t mediaStorageConfig;
+    SignalingResult_t result;
+    const char *message = "{"
+        "\"MediaStorageConfiguration\": {"
+            "\"Status\": \"ENABLED\""
+            // Missing StreamARN
+        "}"
+    "}";
+    size_t messageLength = strlen( message );
+
+    result = Signaling_ParseDescribeMediaStorageConfigResponse( message,
+                                                                messageLength, &( mediaStorageConfig ) );
+
+    TEST_ASSERT_EQUAL( SIGNALING_RESULT_OK, result );
+    TEST_ASSERT_EQUAL_STRING_LEN( "ENABLED", mediaStorageConfig.pStatus, mediaStorageConfig.statusLength );
+    TEST_ASSERT_NULL( mediaStorageConfig.pStreamArn );
+    TEST_ASSERT_EQUAL( 0, mediaStorageConfig.streamArnLength );
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Validate Signaling Parse Describe Media Storage Config functionality.
+ */
+void test_signaling_ParseDescribeMediaStorageConfigResponse( void )
+{
+    SignalingMediaStorageConfig_t mediaStorageConfig;
+    SignalingResult_t result;
+    const char *message  = "{"
+            "\"MediaStorageConfiguration\": {"
+            "\"Status\": \"ENABLED\","
+            "\"StreamARN\": \"arn:aws:kinesisvideo:us-west-2:123456789012:stream/test-stream/1234567890123\""
+            "}"
+            "}";
```

Comment:
```suggestion
    const char *message  = \
        "{"
            "\"MediaStorageConfiguration\":
            {"
                "\"Status\": \"ENABLED\","
                "\"StreamARN\": \"arn:aws:kinesisvideo:us-west-2:123456789012:stream/test-stream/1234567890123\""
            "}"
        "}";
```

Is it possible to format this like this?

Reply:
Its the uncrustify , i tried to remove it from many places on the JSON format, I think I missed it. Thanks for pointing it out, will do in the upcoming PR.